stock_picking_add_transit/README.rst

```diff
@@ -0,0 +1,92 @@
+Stock Picking Add Transit
+=========================
+This module allows to add a transit to an existing stock operation.
+
+.. contents:: Table of Contents
+
+Context
+-------
+In vanilla Odoo, routes have rigid structures.
+For example, when delivering to a customer from a warehouse, you must use each operation defined in your delivery route.
+
+This module mitigates this issue.
+It allows to dynamically insert an internal transfer into your delivery / receipt steps for a given order.
+
+Usage
+-----
+As member of the group ``Stock / User``, I go to the form view of a delivery order.
+
+I notice a new button ``Add Transit``.
+
+.. image:: static/description/delivery_order.png
+
+A wizard is open.
```

Comment:
```suggestion
A wizard is opened.
```

Reply:
L'autre jour, tu m'a fait exactement le commentaire inverse.

Reply:
oh désolé. Les deux peuvent s'écrire c'est tout à fait vrai.

Reply:
En fait, voilà comment je comprends les deux:
`A wizard is open` = un wizard est ouvert
`A wizard is opened` = un wizard s'ouvre.

j'avoue c'est du détail